packages/linwrap/linwrap.9.0.3/opam

```diff
@@ -0,0 +1,85 @@
+opam-version: "2.0"
+authors: "Francois Berenger"
+maintainer: "unixjunkie@sdf.org"
+homepage: "https://github.com/UnixJunkie/linwrap"
+bug-reports: "https://github.com/UnixJunkie/linwrap/issues"
+dev-repo: "git+https://github.com/UnixJunkie/linwrap.git"
+license: "BSD-3"
```

Comment:
```suggestion
license: "BSD-3-Clause"
```

Is ⬆️ the correct one? (see https://spdx.org/licenses/)

Reply:
I guess so